cmd/kube-scheduler/app/server.go

```diff
@@ -111,6 +111,7 @@ for more information about scheduling and the kube-scheduler component.`,
 	cliflag.SetUsageAndHelpFunc(cmd, *nfs, cols)
 
 	if err := cmd.MarkFlagFilename("config", "yaml", "yml", "json"); err != nil {
+		// nolint:logcheck // TODO (?): return error instead of logging it here
 		klog.ErrorS(err, "Failed to mark flag filename")
```

Comment:
can you just use the global logger?

Reply:
Logging shouldn't be used during program init because it hasn't been configured yet, so returning an error and then dealing with that as part of the "normal" program output is better. Having said that, we can also decide that it doesn't matter because this line will never be reached.

Then both nolint or `klog.Background()` are okay. I prefer to decide about the TODO now and lean towards `klog.Background`.